x/tokenfactory/keeper/createdenom_test.go

```diff
@@ -78,24 +86,53 @@ func (suite *KeeperTestSuite) TestCreateDenom() {
 			valid:    false,
 		},
 		{
-			desc:     "success case",
-			subdenom: "evmos",
-			valid:    true,
+			desc:             "success case: defaultDenomCreationFee",
+			denomCreationFee: defaultDenomCreationFee,
+			subdenom:         "evmos",
+			valid:            true,
 		},
 		{
-			desc:     "subdenom having invalid characters",
-			subdenom: "bit/***///&&&/coin",
-			valid:    false,
+			desc:             "success case: twoDenomCreationFee",
+			denomCreationFee: twoDenomCreationFee,
+			subdenom:         "catcoin",
+			valid:            true,
+		},
+		{
+			desc:             "success case: nilCreationFee",
+			denomCreationFee: nilCreationFee,
+			subdenom:         "czcoin",
+			valid:            true,
+		},
+		{
+			desc:             "account doesn't have enough to pay for denom creation fee",
+			denomCreationFee: largeCreationFee,
+			subdenom:         "tooexpensive",
+			valid:            false,
+		},
+		{
+			desc:             "subdenom having invalid characters",
+			denomCreationFee: defaultDenomCreationFee,
+			subdenom:         "bit/***///&&&/coin",
+			valid:            false,
 		},
 	} {
+		suite.SetupTest()
 		suite.Run(fmt.Sprintf("Case %s", tc.desc), func() {
 			if tc.setup != nil {
 				tc.setup()
 			}
-			// Create a denom
+			// Set denom creation fee in params
+			keeper.Keeper.SetParams(*suite.App.TokenFactoryKeeper, suite.Ctx, tc.denomCreationFee)
+			denomCreationFee := suite.App.TokenFactoryKeeper.GetParams(suite.Ctx).DenomCreationFee
```

Comment:
Wdyt about consolitdating all used keepers at the top like:
```
tokenFactoryKeeper := suite.App.TokenFactoryKeeper
...
tokenFactoryKeeper.GetParams(...)
```

It might help with alleviating some of my confusion about what keepers we are setting the parameters on